src/traits.rs

```diff
@@ -780,6 +780,36 @@ pub trait SystemExt: Sized + Debug + Default + Send + Sync {
         self.refresh_cpu_specifics(CpuRefreshKind::new().with_cpu_usage())
     }
 
+    /// Refreshes CPUs frequency information.
+    ///
+    /// Calling this method is the same as calling
+    /// `refresh_cpu_specifics(CpuRefreshKind::new().with_frequency())`.
+    ///
+    /// ```no_run
+    /// use sysinfo::{System, SystemExt};
+    ///
+    /// let mut s = System::new_all();
+    /// s.refresh_cpu_frequency();
+    /// ```
+    fn refresh_cpu_frequency(&mut self) {
+        self.refresh_cpu_specifics(CpuRefreshKind::new().with_frequency())
+    }
+
+    /// Refreshes all information related to CPUs information.
+    ///
+    /// Calling this method is the same as calling
+    /// `refresh_cpu_specifics(CpuRefreshKind::everything())`.
+    ///
+    /// ```no_run
+    /// use sysinfo::{System, SystemExt};
+    ///
+    /// let mut s = System::new_all();
+    /// s.refresh_cpu_all();
+    /// ```
+    fn refresh_cpu_all(&mut self) {
```

Comment:
I think we should rename the existing `refresh_cpu` into `refresh_cpu_usage` and rename this one `refresh_cpu`. What do you think?

Reply:
@GuillaumeGomez done